src/Fabulous.XamarinForms/Views/Controls/Image.fs

```diff
@@ -33,3 +48,17 @@ module ImageBuilders =
 
         static member inline Image<'msg>(stream: System.IO.Stream, aspect: Aspect) =
             View.Image<'msg>(ImageSource.FromStream(fun () -> stream), aspect)
+
+[<Extension>]
+type ImageModifiers =
+    [<Extension>]
+    static member inline aspect(this: WidgetBuilder<'msg, #IImage>, aspect: Aspect) =
```

Comment:
Since Aspect would be mandatory, this extension method can be removed

Reply:
Removed